src/main/java/qna/domain/Answer.java

```diff
@@ -1,15 +1,22 @@
 package qna.domain;
 
+import org.springframework.lang.Nullable;
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.*;
 import java.util.Objects;
 
-public class Answer {
+@Entity
+public class Answer extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private Long writerId;
     private Long questionId;
+    @Lob
     private String contents;
+    @Nullable
     private boolean deleted = false;
```

Comment:
```suggestion
    @Column(name = "deleted", nullable = false)
    private boolean deleted;
```
ddl을 보면 deleted 컬럼을 `not null` 이네요 :)